scripts/webpack/browser.js

```diff
@@ -53,46 +50,54 @@ module.exports = ({
         chunkFilename: generateName(isLegacyJS),
     },
     plugins: [
-        PROD &&
-        new AssetsManifest({
-            writeToDisk: true,
-            assets: isLegacyJS ? legacyManifestData : manifestData,
-            // Need to explicitly define output file names to avoid overwrites
-            output: isLegacyJS ? 'manifest.legacy.json' : 'manifest.json',
+        new LoadablePlugin({
+            filename: isLegacyJS
+                ? 'loadable-stats.browser.legacy.json'
+                : 'loadable-stats.browser.json',
```

Comment:
this is the only change made to the file, the rest are styling changes from prettier